packages/web-components/src/components/content-item-horizontal/__stories__/README.stories.mdx

```diff
@@ -131,6 +150,7 @@ See [our README](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/t
 | ------------------------------------- | -------------------------------------------------- | ----------- |
 | `<dds-content-item-horizontal>`       | `data-autoid="dds--content-item-horizontal"`       | Component   |
 | `<dds-content-item-horizontal-media>` | `data-autoid="dds--content-item-horizontal-media"` | Component   |
+| `<dds-content-item-horizontal-thumbnail>` | `data-autoid="dds--content-item-horizontal-thumbnail"` | Component   |
```

Comment:
This line can be removed as we're no longer defining a component for `<dds-content-item-horizontal-thumbnail>`.